pom.xml

```diff
@@ -15,7 +15,7 @@
         <maven.pmd.plugin.version>3.7</maven.pmd.plugin.version>
         <maven.findbugs.plugin.version>3.0.4</maven.findbugs.plugin.version>
         <maven.pmd.plugin.version>3.8</maven.pmd.plugin.version>
-        <maven.sevntu.checkstyle.plugin.version>1.32.0</maven.sevntu.checkstyle.plugin.version>
+        <maven.sevntu-checkstyle-check.sevntu-checks.version>1.35.0</maven.sevntu-checkstyle-check.sevntu-checks.version>
```

Comment:
Same as other comment. Please revert the property name back to it's original name.

Reply:
done